client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -104,6 +105,7 @@ function ReaderSubscriptionListItem( {
 						className="reader-subscription-list-item__site-url"
 					>
 						{ formatUrlForDisplay( siteUrl ) }
+						{ feed && feed.last_update && translate( ' updated %s ', { args: lastUpdatedDate } ) }
```

Comment:
I'm not sure we should include the spaces either side in the translated string - makes it harder to reuse the same translation somewhere else in Calypso.